projects/1.20-fabric/assets/advanced-xray-fabric-edition/advanced-xray-fabric/lang/zh_cn.json

```
@@ -0,0 +1,55 @@
+{
+	"category.xray": "Advanced XRay",
+	"keybinding.enable_xray": "启用XRay",
+	"keybinding.open_gui": "打开Gui",
+	"message.xray_active": "XRay激活",
+	"message.xray_deactivate": "XRay停用",
+	"spacer": "-------------------------------------------------------",
+	"comment_forge": "Forge mod old lang stuff, please never translate",
+	"xray.single.cancel": "取消",
+	"xray.single.add": "添加",
+	"xray.single.delete": "删除",
+	"xray.single.save": "保存",
+	"xray.single.tools": "工具",
+	"xray.single.close": "关闭",
+	"xray.single.search": "搜索",
+	"xray.single.help": "帮助",
+	"xray.overlay": "XRay激活",
+	"xray.color.red": "红：",
+	"xray.color.green": "绿：",
+	"xray.color.blue": "蓝：",
+	"xray.input.gui": "GUI的名字",
+	"xray.input.add": "添加方块",
+	"xray.input.add_hand": "添加手里的方块",
+	"xray.input.add_look": "添加查看的地方",
+	"xray.input.distance": "距离：%s",
+	"xray.input.show-lava": "显示熔岩：%s",
+	"xray.input.toggle_oredict": "使用字典",
+	"xray.title.config": "配置方块",
+	"xray.title.edit": "编辑方块",
+	"xray.title.edit_meta": "编辑元数据",
+	"xray.message.missing_input": "您需要填写所有输入框",
```

```suggestion
	"xray.message.missing_input": "你必须填写所有输入项",
```
一般不用“您”。
所有输入一定是在“框”中输入的吗？